SPECS/intel-i40e-alt.spec

```diff
@@ -64,6 +62,12 @@ find %{buildroot}/lib/modules/%{kernel_version} -name "*.ko" -type f | xargs chm
 /lib/modules/%{kernel_version}/*/*.ko
 
 %changelog
+* Fri Nov 15 2024 David Morel <david.morel@vates.tech> - 2.26.8-1
+- Update from Intel's latest driver 2.26.8
+- Removed NEED_AUX_BUS=2 that tries to detect if auxiliary informations are
```

Comment:
`NEED_AUX_BUS=2` is actually used to compile the auxiliary bus code within the i40e driver. We don't need it since our kernel already provides that auxiliary bus driver.
And `NEED_AUX_BUS=1` can be used to print the info about the auxiliary module of the build env.

Reply:
Rewrote that, thank you for clarifying it.